docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md

```diff
@@ -0,0 +1,122 @@
+To see the users that are assigned a specific role, navigate to **Admin > Access Management > Roles**, click **Actions** for the role, and choose **View Assigned Users**.
+
+### Built-in roles
+
+The following built-in roles are available:
+
+* **Super Admin** is the first user that is created during the installation. This role has the highest level of privilege that allows all read and write actions on all YugabyteDB Anywhere resources. There can be only one Super Admin. This Super Admin can perform the following:
+
+  * Manage all resources, including universes, nodes, backup, restore, and cloud providers.
+  * Manage the user access control by creating and managing users.
+
+  For more information, see [admin user](../../configure-yugabyte-platform/create-admin-user/).
+
+* **Admin** has privileges that are similar to the Super Admin, except that the Admin cannot manage the global scope artifacts and actions, such as runtime configuration settings and LDAP authentication.
+
+* **Backup Admin** has access to the backup related tasks, such as the following:
+
+  * Manage database backups and restore operations.
+  * Create new backups.
+  * Delete any existing backup or backup schedule.
+  * Edit existing backups.
+  * Read-only permissions for all other resources in YugabyteDB Anywhere.
+
+* **Read-only** access level provides viewer permission to the UI and API.
+
+You can't delete or edit built-in roles.
+
+### Create, modify, and delete roles
+
+As a Super Admin or Admin, you can:
+
+* create custom roles
+* clone built-in and custom roles
+* modify and delete custom roles
+
+To create a custom role, do the following:
+
+1. Navigate to **Admin > Access Management > Roles** and click **Create Role**.
+1. Enter a name for the role.
+1. Enter a description for the role.
+1. Click **Select Permissions**.
+1. Select the permissions to assign to the role and click **Confirm** when you are done.
+1. Click **Save**.
+
+To create a custom role from an existing role, do the following:
+
+1. Navigate to **Admin > Access Management > Roles**, click **Actions** for the role to clone, and choose **Clone Role**.
+1. Enter a name for the role.
+1. Enter a description for the role.
+1. Click **Edit Permissions**.
+1. Select the permissions to assign to the role and click **Confirm** when you are done.
+1. Click **Save**.
+
+To edit a custom role, do the following:
+
+1. Navigate to **Admin > Access Management > Roles**, click **Actions** for the role to edit, and choose **Edit Role**.
+1. On the **Configurations** tab click **Edit Permissions**.
+1. Select the permissions to assign to the role and click **Confirm** when you are done.
+1. Click **Save**.
+
+To delete a role, navigate to **Admin > Access Management > Roles**, click **Actions** for the role to edit, and choose **Edit Role**.
+
+To view the users that have been assigned a role, navigate to **Admin > Access Management > Roles**, click **Actions** for the role, and choose **View Assigned Users**.
```

Comment:
@ddhodge @vpatibandla-yb Should we also add here some of the limitations (from our internal design doc) that we have in RBAC phase 1, which will be solved in phase 2?